js/widgets/forms/clearButton.js

```diff
@@ -20,59 +25,56 @@ define( [
 		_create: function() {
 			this._super();
 
+			if ( !this.inputNeedsWrap ) {
+				return;
+			}
```

Comment:
Unrelated to this PR autogrow should have its own Boolean option and this should all be wrapped in a conditional.